src/com/musala/xml/model/School.java

```diff
@@ -0,0 +1,54 @@
+package com.musala.xml.model;
+
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "school")
+public class School {
```

Comment:
Please insert a blank line to separate the class declaration from the declaration of the class properties (this is a good visual practice).